### PR TITLE
Adds a button to enable voice chat in Secrets.

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball, Cheridan
add: Adds voice chat to the game.
/:cl:

Demonstration here:
https://file.house/HHgJ.webm

### CODE REVIEW

**Comment by chesse20** [comment]
merge this NAOW no review!

File: code/_globalvars/misc.dm [hunk 1]
```diff
@@ -17,4 +17,7 @@ GLOBAL_LIST_EMPTY(powernets)
 
 GLOBAL_VAR_INIT(bsa_unlock, FALSE)	//BSA unlocked by head ID swipes
 
+GLOBAL_VAR_INIT(verbose_speech, FALSE) // VOCAL CORD PARASITES
```
Comment (chesse20) [inline code/_globalvars/misc.dm]: can we have this be true by default

**Comment by Iamgoofball** [comment]
>>>travis
chrome_2018-06-30_03-41-16

**Comment by optimumtact** [comment]
goof the next pr you make that isn't worth merging I will github ban you forever

**Comment by outbools** [comment]
I CAN SEE THAT LONE STAR FROM THE THOUSAND MILES AWAY

**Comment by chesse20** [comment]
who gave optimumtact member they just keep closing good prs

**Comment by PKPenguin321** [comment]
meme MEME meme

**Comment by GuyonBroadway** [comment]
Memes aside, this coulda been amusing. Imagine if the sound worked like how bombs have the distant bomb effect and you could hear muffled talking a distance away or behind doors and stuff.

And whispering would make a noise, and yelling too.

It might make stations feel more alive with the gentle hum of conversation.

**Pull request closed without merging**